### ISSUE TITLE
Rifle scope clickers

### ISSUE THREAD

**Issue by runnybunny** [KoffeinFlummi/AGM#2019]
The clickers that show up at the top of the screen show up less than half of the time. We have to rely on counting the clicks.

**Comment by Ataraxic89** [comment]
They show up whenever you click it. But only when not zoomed. 

This is how you use it: Learn how your weapon fires by extended range practice. Learn how many clicks it takes to zero at 300m, 600m, 900m, etc and all between. 

Once in game use your range finder to figure your targets distance. Now zero your rifle. Remember to account for angle, exact range (653m isnt a common practice range), and other effects. 

Squeeze the trigger. If you miss (you didnt practice enough) you can quickly adjust up or down without unzooming. This should only take 1-5 clicks and its more about feel anyway so you dont need to see the clickers. Now hit your target. 

TL;DR: not an issue.

**Comment by commy2** [comment]
>They show up whenever you click it. But only when not zoomed. 

This.
